lib/truffle/truffle/patching.rb

```diff
@@ -0,0 +1,38 @@
+module Truffle::Patching
+  TRUFFLE_PATCHES_DIRECTORY = File.join(Truffle::Boot.ruby_home, 'lib', 'patches')
+  TRUFFLE_PATCHES           = Dir.glob(File.join(TRUFFLE_PATCHES_DIRECTORY, '**', '*.rb')).
```

Comment:
Please avoid `File.join`, it just adds clutter, and in the case of `Dir.glob` is even potentially incorrect (if File::SEPARATOR is changed). Filesystem pathnames in Ruby are always represented with `/`.